src/index/index.react-server.ts

```diff
@@ -1 +1,2 @@
 export { unstable_serialize } from '../core/serialize'
+export { SWRConfig } from './config'
```

Comment:
Should we expose it to server components ?

Since <SWRConfig /> can take function as props.

Reply:
yeah we decide to expose it now, cannot pass down function is like a general issue now that hard to get rid of

Reply:
Users can create a new client boundary to work around this, so might be a special case / documentation issue.